### ISSUE TITLE
AutomationProperties.HelpText not read by Narrator

### ISSUE THREAD

**Issue by ahopper** [AvaloniaUI/Avalonia#16999]
### Describe the bug

If AutomationProperties.HelpText is set on a button the windows screen reader Narrator does not read the text, it does read it on a wpf program with same property.
[AvaloniaAccess.zip](https://github.com/user-attachments/files/16978696/AvaloniaAccess.zip)

### To Reproduce

Run the sample on windows, start Narrator (windows key ctrl enter) and tab to the first button.

### Expected behavior

the help text to be read out

### Avalonia version

11.2.999-cibuild0051813-alpha 11.1.3

### OS

Windows
